### PR TITLE
Docs code templates

### PR DESCRIPTION
**Pull request by agriffard**
Documentation to explain how to specify the Framework with `dotnet new` command line.

Related to #8695

### CODE REVIEW

File: src/docs/getting-started/templates/README.md [hunk 1]
```diff
@@ -47,6 +51,12 @@ Options:
                          Default: 1.0.0-rc2
 ```
 
+The `3.1` .NET Core Framework can be used with this command:
+
+```CMD
+dotnet new occms --Framework netcoreapp3.1
```
Comment (jtkech) [inline src/docs/getting-started/templates/README.md:57]: `--framework` with a lower case

File: src/docs/getting-started/templates/README.md [hunk 2]
```diff
@@ -36,7 +36,11 @@ You can pass the following CLI parameters to setup options:
 Orchard Core Cms Web App (C#)
 Author: Orchard Project
 Options:
-  -lo|--logger           Configures the logger component.
+  --Framework            Configures the Framework.
```
Comment (jtkech) [inline src/docs/getting-started/templates/README.md:39]: `--framework` with a lower case

**Comment by agriffard** [comment]
Why the .template.config files were ignored?

Do you manage to have the OC templates when you create a new project from VS?

File: src/Templates/OrchardCore.ProjectTemplates/content/OrchardCore.Templates.Cms.Module/.template.config/dotnetcli.host.json [hunk 3]
```diff
@@ -0,0 +1,16 @@
+{
+  "$schema": "http://json.schemastore.org/dotnetcli.host",
+  "symbolInfo": {
+    "Framework": {
+      "longName": "framework",
+      "shortName": "fk"
```
Comment (jtkech) [inline src/Templates/OrchardCore.ProjectTemplates/content/OrchardCore.Templates.Cms.Module/.template.config/dotnetcli.host.json:6]: I would have used `fm`

File: src/Templates/OrchardCore.ProjectTemplates/content/OrchardCore.Templates.Cms.Module/.template.config/dotnetcli.host.json [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    },
+    "OrchardVersion": {
+      "longName": "orchard-version",
+      "shortName": "ov"
+    }
+  },
+  "usageExamples": [
+    "--orchard-version 1.0.0-beta3"
+  ]
+}
```
Comment (jtkech) [inline src/Templates/OrchardCore.ProjectTemplates/content/OrchardCore.Templates.Cms.Module/.template.config/dotnetcli.host.json:16]: Missing new line

File: src/Templates/OrchardCore.ProjectTemplates/content/OrchardCore.Templates.Cms.Web/.template.config/dotnetcli.host.json [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+    "OrchardVersion": {
+      "longName": "orchard-version",
+      "shortName": "ov"
+    }
+  },
+  "usageExamples": [
+    "--nlog false",
+    "--orchard-version 1.0.0-rc2"
+  ]
+}
```
Comment (jtkech) [inline src/Templates/OrchardCore.ProjectTemplates/content/OrchardCore.Templates.Cms.Web/.template.config/dotnetcli.host.json:21]: New line

File: src/Templates/OrchardCore.ProjectTemplates/content/OrchardCore.Templates.Cms.Web/.template.config/template.json [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+          ]
+        }
+      ]
+    }
+  ],
+  "tags": {
+    "language": "C#",
+    "type": "project"
+  }
+}
```
Comment (jtkech) [inline src/Templates/OrchardCore.ProjectTemplates/content/OrchardCore.Templates.Cms.Web/.template.config/template.json:81]: New line

File: src/Templates/OrchardCore.ProjectTemplates/content/OrchardCore.Templates.Mvc.Module/.template.config/dotnetcli.host.json [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+    },
+    "OrchardVersion": {
+      "longName": "orchard-version",
+      "shortName": "ov"
+    }
+  },
+  "usageExamples": [
+    "--orchard-version 1.0.0-beta3"
+  ]
+}
```
Comment (jtkech) [inline src/Templates/OrchardCore.ProjectTemplates/content/OrchardCore.Templates.Mvc.Module/.template.config/dotnetcli.host.json:16]: New line

File: src/Templates/OrchardCore.ProjectTemplates/content/OrchardCore.Templates.Mvc.Web/.template.config/dotnetcli.host.json [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+    },
+    "OrchardVersion": {
+      "longName": "orchard-version",
+      "shortName": "ov"
+    }
+  },
+  "usageExamples": [
+    "--orchard-version 1.0.0-beta3"
+  ]
+}
```
Comment (jtkech) [inline src/Templates/OrchardCore.ProjectTemplates/content/OrchardCore.Templates.Mvc.Web/.template.config/dotnetcli.host.json:16]: New line

File: src/Templates/OrchardCore.ProjectTemplates/content/OrchardCore.Templates.Mvc.Web/.template.config/template.json [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+      "defaultValue": "1.0.0-rc2"
+    }
+  },
+  "sources": [
+  ],
+  "tags": {
+    "language": "C#",
+    "type": "project"
+  }
+}
```
Comment (jtkech) [inline src/Templates/OrchardCore.ProjectTemplates/content/OrchardCore.Templates.Mvc.Web/.template.config/template.json:43]: New line

File: src/Templates/OrchardCore.ProjectTemplates/content/OrchardCore.Templates.Theme/.template.config/template.json [hunk 10]
```diff
@@ -0,0 +1,62 @@
...
+          ]
+        }
+      ]
+    }
+  ],
+  "tags": {
+    "language": "C#",
+    "type": "project"
+  }
+}
```
Comment (jtkech) [inline src/Templates/OrchardCore.ProjectTemplates/content/OrchardCore.Templates.Theme/.template.config/template.json:62]: New line

**Pull request merged**